chapters/en/slides/module5/module5_04.Rmd

```diff
@@ -0,0 +1,348 @@
+to separate the points belonging to different categories.
+We can quickly identify that there is a tota of three distinct colors being used here
+even without looking at the legend.
+
+In the plot to the right
+we used lightness within a single hue (blue) to label the different categories.
+Here it is much harder to say how many different categories there are,
+and even when we have the legend,
+it is difficult to tell which points are which shade of blue.
+
+---
+
+## It is often better to use established color schemes instead of making your own
+
+<center><img src=/module5/color-schemes.png></img></center>
+
+Notes:
+There are many different hues we could use to represent our colors
+and some might not go as well together as others,
+so how do we decide which ones to use?
+
+Fortunately,
+we don’t have to design our own combination of colors to use,
+but can pick from combinations designed by experts to be easy to tell apart
+and in most cases also suitable for people with color vision deficiencies.
+
+These color combinations are referred to as color schemes in Altair,
+but you might also hear them being called colormaps or color palettes.
+
+In this slide, you can see some of the color schemes for categorical values
+that are built into Altair.
+
+The default one is “tableau10”
+and it was named this way because it was originally designed by the company Tableau,
+but is now one of the most common categorical color schemes used in data visualization.
+[All Altair color schemes can be viewed here](https://vega.github.io/vega/docs/schemes/).
+
+---
+
+## Specifying color schemes in Altair
+
+```{python}
+alt.Chart(cars).mark_point(size=70, filled=True).encode(
+    x='Horsepower',
+    y='Miles_per_Gallon',
+    color=alt.Color('Origin', title=None, scale=alt.Scale(scheme='set1')))
+```
+
+Notes:
+You can change the color scheme of a plot by specifying its name as a string
+to the `scheme` parameter inside `alt.Scale`.
+
+To easier see the changes we are making to the colors,
+we also increase the size of all points 
+via the `size` parameter of the mark.
+
+---
+
+## Redundant coding can make charts easier to interpret
+
+```{python}
+alt.Chart(cars).mark_point(size=70, filled=True).encode(
+    x='Horsepower',
+    y='Miles_per_Gallon',
+    color=alt.Color('Origin', title=None, scale=alt.Scale(scheme='set1')),
+    shape='Origin')
+```
+
+Notes:
+Although many of the color schemes in Altair 
+are designed according to the guidelines for effective color uses,
+they can sometimes be difficult to interpret 
+for people with color vision deficiences.
+
+Esppecially color schemes that mix red and green
+as the one in the previous slide.
+
+In addition to using a more suitable color scheme (such as the default),
+we could also change the shape of the points for each category.
+
+Although this is technically redundant
+since the color is already used for the categorical groups,
+it can make your visualization more effective.
```

Comment:
Why? Because the marks "pop out" as being different, and this is helpful for folks who do not have a disability as well as those who do.